sigstore/_cli.py

```diff
@@ -329,6 +329,12 @@ def _parser() -> argparse.ArgumentParser:
         default=_boolify_env("SIGSTORE_OVERWRITE"),
         help="Overwrite preexisting signature and certificate outputs, if present",
     )
+    output_options.add_argument(
```

Comment:
I am unsure if this belongs to `output_options`, I can change that if necessary

Reply:
`output_options` works for me! It could also go under a new `signing_options` or similar, but no strong opinion there.